source/qb64pe.bas

```diff
@@ -24572,6 +24574,34 @@ SUB Set_OrderOfOperations
     i = i + 1: OName(i) = "_BLUE": PL(i) = 10
     i = i + 1: OName(i) = "_ALPHA": PL(i) = 10
 
+    Skip_NoPrefix_Low = i + 1
+    i = i + 1:: OName(i) = "PI": PL(i) = 10
```

Comment:
Is it really better to list all of these out separately rather than stripping the `_` during the check like we were before? It just seems like this is harder to maintain since we now have to make sure this list matches all the `_` entries listed above.

Reply:
I'd think so.  It should speed up the overall process of handling things, as it just skips these values if noprefix isn't in use, compared to previously where it ran those loops reagardless.  It's a little more code here, but less processing there.

Reply:
We're only talking about 20 or so strings, why is it worth it to make the code harder to maintain when the speed-up is presumably not noticeable?

I'm not so sure it's actually faster either, the `Select Case OName()`s now have to check for both the prefixed and non-prefixed versions and that's not conditional on `qb64prefix_set`.